install-files/PyMicroService/astro_time_convert.py

```diff
@@ -0,0 +1,67 @@
+##############################################################################
+# astro_time_convert.py
+##############################################################################
```

Comment:
Do we need `requirements.txt` or similar to make it easier for users of this micro service to have the right dependencies and Python version installed?